### PR TITLE
Create 2021-12-10-recap-code-for-germany-community-summit-2021

### PR DESCRIPTION
**Pull request by sonjafischbauer**
Blogpost to recap the recent event of the Code for Germany Community Summit 2021.
I apologize in advance for all mistakes, I did my best.

### CODE REVIEW

**Comment by ubergesundheit** [comment]
I think you forgot to add the picture

File: content/blog/2021-12-10-recap-code-for-germany-community-summit-2021 [hunk 1]
```diff
@@ -0,0 +1,50 @@
+---
```
Comment (ubergesundheit) [inline content/blog/2021-12-10-recap-code-for-germany-community-summit-2021:1]: The file extension should be .md

File: content/blog/2021-12-10-recap-code-for-germany-community-summit-2021 [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+title: "Das war der Code for Germany Summit 2021"
+author: "Sonja Fischbauer"
+date: '2021-12-31T17:20:00+01:00'
```
Comment (ubergesundheit) [inline content/blog/2021-12-10-recap-code-for-germany-community-summit-2021:4]: Any reason to date this to the end of the year?

File: content/blog/2021-12-10-recap-code-for-germany-community-summit-2021 [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+
+Neben einem kleinen Meetup im Rahmen des [Hackday Moers](https://www.codeforniederrhein.de/hackday-2021/) war der Summit das erste bundesweite vor-Ort-Treffen der Code for Germany Community seit der [Erstauflage im Jahr 2019](https://codefor.de/blog/cfg-community-summit-2019/). 
+
+Dieses Jahr kamen insgesamt 16 Teilnehmende, davon 13 Ehrenamtliche am Samstag, 27. November in den Räumen des bUm am Paul-Lincke-Ufer in Berlin unter 2G+ Regelung zusammen. 
+
+Am Abend davor hatte ein Großteil der Leute schon die Möglichkeit genutzt und sich bei Pizza, Limo und anderen Drinks getroffen. Am Tag des Summits starteten wir Vormittags mit einer ausgiebigen Vorstellungsrunde und Berichten aus den lokalen Labs: Ehrenamtliche Communitymitglieder waren aus Hamburg, Berlin, Köln, Moers/ Niederrhein, Leipzig, Chemnitz, Kaiserslautern, Nürnberg und Ulm angereist! Die angeregten Gespräche über die innere Struktur und die äußere Wirkung von Code for Germany begleiteten uns bis über die Mittagspause.
+
+  - imgname: cfgsummit21-austausch.jpg
```
Comment (ubergesundheit) [inline content/blog/2021-12-10-recap-code-for-germany-community-summit-2021:25]: What happened here? I think this line needs to be removed

Comment (ubergesundheit) [reply content/blog/2021-12-10-recap-code-for-germany-community-summit-2021:25]: Or do you want to display the picture here? In this case , I think the syntax is `description`

**Comment by ubergesundheit** [comment]
You can check the deploy preview here: https://deploy-preview-310--hungry-heisenberg-4bd67f.netlify.app/

your post is not there because of multiple things: 
- the file extension is not .md
- Maybe the date is in the future

File: content/blog/2021-12-10-recap-code-for-germany-community-summit-2021.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+Wer den Code einsehen - oder auch daran mitarbeiten - möchte, findet das [Open Source Projekt auf Github](https://github.com/okfde/koalitionstracker).
+
+![Foto: In einer Arbeitsgruppe kartierten wir Wirkungssphären und gesellschaftliche Ziele unseres Netzwerks. Foto von Wolfram Eberius, Lizenz [CC-BY 4.0]](recap-code-for-germany-community-summit-2021/cfgsummit21-mapping.jpg)
+
+Neben der praktischen Umsetzung beschäftigten wir uns auch mit den politischen und gesellschaftlichen Zielen unseres Netzwerks. Wie wirken die Open Knowledge Labs? Auf welche gesellschaftlichen Ziele arbeitet Code for Germany hin? Diesen Fragen widmeten wir uns in einer kollaborativen Arbeitsgruppe. Wir kartierten wir die Angebote und Wirkungsspähren von Code for Germany, sowohl auf der Ebene der regionalen Labs als auch des bundesweiten Netzwerks.
+
+Zwischen all diesen Aufgaben haben wir uns auf dem Summit viel Zeit genommen um miteinander zu reden, zu diskutieren und endlich einmal wieder vor Ort zusammen zu kommen. In kleiner Runde mit 2G+ hatten wir entspannte Stimmung und alle blieben soweit gesund!
```
Comment (ubergesundheit) [inline content/blog/2021-12-10-recap-code-for-germany-community-summit-2021.md:33]:
```suggestion
Zwischen all diesen Aufgaben haben wir uns auf dem Summit viel Zeit genommen um miteinander zu reden, zu diskutieren und endlich einmal wieder vor Ort zusammen zu kommen. In kleiner Runde mit 2G+ hatten wir entspannte Stimmung und alle blieben soweit gesund!

![Foto: Beim Summit entstand der Koalitionstracker, ein Tool das die Umsetzung der neuen Koalitionsvorhaben nachhält.
Foto von Wolfram Eberius, Lizenz [CC-BY 4.0]](recap-code-for-germany-community-summit-2021/cfgsummit21-koalitionstracker.jpg)
```

File: content/blog/2021-12-10-recap-code-for-germany-community-summit-2021.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+
```
Comment (sonjafischbauer) [inline content/blog/2021-12-10-recap-code-for-germany-community-summit-2021.md:34]:
```suggestion
  - imgname: cfgsummit21-koalitionstracker.jpg
Foto: Beim Summit entstand der Koalitionstracker, ein Tool das die Umsetzung der neuen Koalitionsvorhaben nachhält.
Foto von Wolfram Eberius, Lizenz [CC-BY 4.0](https://creativecommons.org/licenses/by/4.0/)
```

File: content/blog/2021-12-10-recap-code-for-germany-community-summit-2021.md [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+![Foto: Beim Summit entstand der Koalitionstracker, ein Tool das die Umsetzung der neuen Koalitionsvorhaben nachhält.
+Foto von Wolfram Eberius, Lizenz [CC-BY 4.0]](recap-code-for-germany-community-summit-2021/cfgsummit21-koalitionstracker.jpg)
```
Comment (ubergesundheit) [inline content/blog/2021-12-10-recap-code-for-germany-community-summit-2021.md:36]:
```suggestion
![Foto: Beim Summit entstand der Koalitionstracker, ein Tool das die Umsetzung der neuen Koalitionsvorhaben nachhält. Foto von Wolfram Eberius, Lizenz [CC-BY 4.0]](recap-code-for-germany-community-summit-2021/cfgsummit21-koalitionstracker.jpg)
```

File: content/blog/2021-12-10-recap-code-for-germany-community-summit-2021.md (hunk 5 above)
Comment (sonjafischbauer) [reply content/blog/2021-12-10-recap-code-for-germany-community-summit-2021.md:34]: Foto:  Beim Summit entstand der Koalitionstracker, ein Tool das die Umsetzung der neuen Koalitionsvorhaben nachhält. Foto von Wolfram Eberius, Lizenz [CC-BY 4.0] (cfgsummit21-koalitionstracker.jpg)

**Review by ubergesundheit — APPROVED** [review]
Looks good :)

**Pull request merged**